docs/UserGuide.md

```diff
@@ -915,19 +971,20 @@ Format: `takeattendance n/STUDENT_NAME s/SESSION_NUMBER ap/PRESENCE`
 
 You can view the attendance list of your students.
 
-Format: `viewattendance n/STUDENT_NAME n/STUDENT_NAME …`
-
-| Parameter    | Description                                                    |
-|--------------|----------------------------------------------------------------|
-| STUDENT_NAME | The name of the student(s) you want to view the attendance of. |
+Format: `viewattendance [n/STUDENT_NAME]…`
 
+| <center>**Parameter**</center> | <center>**Description**</center>                                                |
+|--------------------------------|---------------------------------------------------------------------------------|
+| <center>STUDENT_NAME</center>  | <center>The name of the student(s) you want to view the attendance of.</center> |
+<br>
 
 <div class="alert alert-info"> 
 <md>
 
 :information_source: Note:
 
 * The student must exist in F.A.K.E.J.A.R.V.I.S.
+* Leaving the `STUDENT_NAME` field blank will display the overall attendance across all students and sessions. 
```

Comment:
Omitting `n/STUDENT_NAME` will display the overall attendance across all students and sessions. (?)

Reply:
roger